package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
```

Comment:
How about we have a script like `"dev-start": "APP_ENV=development node index",` and leave `"start": "node index",`